pages/index.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-layout justify-center column align-content-center>
     <h2>{{ $t('welcome') }}</h2>
-    <v-card itemscope itemtype="http://schema.org/GovernmentOrganization">
+    <v-card itemscope :itemtype="governmentOrganization">
```

Comment:
Oh, I liked to use v-bind with static strings to make URLs more readable. It happens in a lot of places, like `:href="gawURL"` or `:href="wmoURL"` instead of big long static strings.

Given what you say about v-bind with non-dynamic strings, should all these URLs be put straight in the href rather than being referenced by variables?

Reply:
Generally, I agree with your approach and I too would go with the v-bind route like you did with the `gawURL`, especially when there are multiple instances of links that use the same URL. 

It's just the microdata attributes (`itemtype`, `itemscope`, etc...) that I'm unsure of if v-binding  may affect the SEO; better to play it safe and keep that static.